src/main/java/domain/result/LottoRank.java

```diff
@@ -3,12 +3,12 @@
 import java.util.Arrays;
 
 public enum LottoRank {
-    NONE_MATCHES(0, 0, false),
-    THREE_MATCHES(5_000, 3, false),
-    FOUR_MATCHES(50_000, 4, false),
-    FIVE_MATCHES(1_500_000, 5, false),
-    FIVE_AND_BONUS_MATCHES(30_000_000, 5, true),
-    SIX_MATCHES(2_000_000_000, 6, false);
+    NO_PRIZE(0, 0, false),
+    FIFTH_PRIZE(5_000, 3, false),
+    FOURTH_PRIZE(50_000, 4, false),
+    THIRD_PRIZE(1_500_000, 5, false),
+    SECOND_PRIZE(30_000_000, 5, true),
+    FIRST_PRIZE(2_000_000_000, 6, false);
```

Comment:
```suggestion
    NO_PRIZE(0, 0),
    FIFTH_PRIZE(5_000, 3),
    FOURTH_PRIZE(50_000, 4),
    THIRD_PRIZE(1_500_000, 5),
    SECOND_PRIZE(30_000_000, 5, true),
    FIRST_PRIZE(2_000_000_000, 6);
```

```
LottoRank(final int prizeMoney, final int lottoBallMatch) {
    this(prizeMoney, lottoBallMatch, false);
}
```

생성자를 추가하면 중복을 제거할 수 있을 것 같습니다

Reply:
네! 반영하겠습니다~